### PR TITLE
feature/hero-update 

### PR DESCRIPTION
**Pull request by Sonny97**
hi y'll the new design in the home has been for you.
watch that and tell me what you reckon.
Screenshot_2019-05-22 Hostal las paisas
quite the last title and put the new title with the button to go down

Also include
fixed the error about the title in title carrousel
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,14 +18,14 @@
 		<div class="homepage-hero__layer"></div>
 		<article class="homepage-hero__article">
```
Comment (josemariogutierrez) [inline index.html]: You can get rid off this article wrapper div. Also i recomend to rename the classes according their content. For example instead of `homepage-hero__headline` you could use `homepage-hero__hdlp-logo`.

File: css/home.css [hunk 2]
```diff
@@ -69,21 +69,30 @@ body {
...
 .homepage-hero__article {
-  margin-right: 5%;
-  margin-left: 5%;
+  margin-right: 57px;
+  margin-left: 57px;
+  position: absolute;
+  bottom: 250px;
+  top: 250px;
+}
+.homepage-hero__lookdown{
```
Comment (josemariogutierrez) [inline css/home.css]: Please look how to center absolute positioned elements, this icon is not centered in dektop view. Also search how to center the logo vertically and horizontally.

**Comment by jhonatanrojasbastidas** [comment]
index.js say you have conflict files.

File: css/home.css (hunk 2 above)
Comment (hanna446) [reply css/home.css]: Please add a space between the selector and the bracket.

File: css/home.css [hunk 3]
```diff
@@ -69,21 +69,30 @@ body {
...
 .homepage-hero__headline {
-  margin-bottom: 5px;
+  width: 247px;
+  height: 140px;
+  object-fit: contain;
+  /*margin-bottom: 5px;
   color: #ffffff;
   font-size: 36px;
-  font-weight: bold;
+  font-weight: bold;*/
```
Comment (hanna446) [inline css/home.css]: I think you should delete this comment.

File: js/index.js [hunk 4]
```diff
@@ -46,3 +46,9 @@ const carrouselImage = document.querySelector(".carrousel__img");
 carrouselArrows.forEach(function(element) {
   element.addEventListener("click", carrousel);
 });
+/* Function about the arrow down or drop down wherever you want
+*/
+function goDown(){
+  var lookdown = document.getElementById('carrousel-section');//take the destini to scroll
```
Comment (hanna446) [inline js/index.js]: please remember the structure of comments.

Comment (hanna446) [reply js/index.js]: I think you shouldn't use **var** as a good practice, you can use **let.**
I also think we agreed that we would use **arrow functions**.
Please, if I'm wrong, correct me.

File: index.html [hunk 5]
```diff
@@ -15,17 +15,17 @@
...
-			<div class="homepage-hero__divider"></div>
-			<span id="" class="homepage-hero__promotext">El paraiso te espera</span>
-		</article>
+    <div class="homepage-hero__layer"></div>
+    <div class="homepage-hero__hdlp">
+      <div class="homepage-hero__hdlp-logo">
+        <img src="assets/icons/logo-hdlp.svg">
+      </div>
+    </div>
+    <div class="homepage-hero__lookdown" id="hero-lookdown" onclick="goDown()">
```
Comment (camigomezdev) [inline index.html]: please don't add the event inline, Use addEventListener. 😅

File: css/home.css [hunk 6]
```diff
@@ -69,21 +70,32 @@ body {
 }
 
 .homepage-hero__article {
-  margin-right: 5%;
-  margin-left: 5%;
+  margin: auto;
+}
+
+.homepage-hero__hdlp-logo{
```
Comment (camigomezdev) [inline css/home.css]: what is HDLP?

File: css/home.css [hunk 7]
```diff
@@ -69,21 +70,32 @@ body {
...
 
-.homepage-hero__headline h1 {
-  margin: 0;
+.homepage-hero__hdlp{
+  margin-left: 50%;
+  position: absolute;
+  bottom: 20px;
+}
+
+.homepage-hero__hdlp-logo h1 {
```
Comment (camigomezdev) [inline css/home.css]: Please, Don't use the tag to add styles, use classes.

File: css/home.css [hunk 8]
```diff
@@ -69,21 +70,32 @@ body {
...
-  font-weight: bold;
+.homepage-hero__lookdown{
+  margin-left: 50%;
+  position: absolute;
+  bottom: 20px;
 }
 
-.homepage-hero__headline h1 {
-  margin: 0;
+.homepage-hero__hdlp{
```
Comment (camigomezdev) [inline css/home.css]: Add a space between the class name and `{`

Comment (camigomezdev) [reply css/home.css]: Give format to all document.

**Comment by jhonatanrojasbastidas** [comment]
Tienes conflictos.

**Review by jhonatanrojasbastidas — COMMENTED** [review]
Conflicts with js file.

File: css/home.css (hunk 6 above)
Comment (josemariogutierrez) [reply css/home.css]: "Hostal donde las paisas"

Comment (camigomezdev) [reply css/home.css]: Maybe this should be `.homepage-hero__logo` and if you have a particular style for the Hostal logo, you should create other class with a modifier `.homepage-hero__logo--hdlp`

Comment (jusk02) [reply css/home.css]: +45262847 @camigomez35

File: index.html [hunk 9]
```diff
@@ -15,13 +15,13 @@
 	</nav>
 	<section class="homepage-hero">
 		<!-- Hero section, the cover for the page -->
-		<div class="homepage-hero__layer"></div>
-		<article class="homepage-hero__article">
-			<div class="homepage-hero__headline">
+    <div class="homepage-hero__layer"></div>
+    <div class="homepage-hero__hdlp">
```
Comment (josemariogutierrez) [inline index.html]: The logo don't need a wrapper div to align it. You can remove it and center de logo inside the section.

File: css/home.css [hunk 10]
```diff
@@ -69,29 +70,31 @@ body {
...
+}
+
+.homepage-hero__hdlp-logo{
   position: absolute;
   bottom: 250px;
-  top: 250px;
+  margin-left: -125px;
+  width: 247px;
+  height: 140px;
+  object-fit: contain;
```
Comment (josemariogutierrez) [inline css/home.css]: The logo and the `lookdown` icon are not being centered correctly, don't use fixed values like `bottom: 250px`. To center an absolute positioned element horizontally you could use for example `margin: auto` and `0` value for `right` and `left` positions. To also center vertically you add `0` value for `top` and `bottom` positions as well.

Comment (josemariogutierrez) [inline css/home.css]: Remove `object-fit: contain` is not necessary here.

File: css/home.css [hunk 11]
```diff
@@ -69,29 +70,31 @@ body {
...
-  margin-left: 57px;
+  margin: auto;
+}
+
+.homepage-hero__hdlp-logo{
   position: absolute;
   bottom: 250px;
-  top: 250px;
+  margin-left: -125px;
+  width: 247px;
```
Comment (josemariogutierrez) [inline css/home.css]: Remember to add the size of the logo for desktop, which is `350px`. Add it inside the already existing media querie for desktop.

File: assets/icons/look-down.svg [hunk 12]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="34" height="40" viewBox="0 0 34 40">
+    <path fill="#FFF" fill-rule="nonzero" d="M18.865 39.15a2.45 2.45 0 0 1-3.447-.01L1.63 25.544a2.366 2.366 0 0 1 0-3.389l2.301-2.259a2.447 2.447 0 0 1 3.437 0l9.774 9.637 9.783-9.627a2.447 2.447 0 0 1 3.437 0l2.291 2.26c.953.93.953 2.449 0 3.388L18.865 39.15zm-2.27-28.606c-2.872 0-5.2-2.36-5.2-5.272 0-2.912 2.328-5.272 5.2-5.272 2.87 0 5.198 2.36 5.198 5.272 0 2.912-2.328 5.272-5.199 5.272z"/>
```
Comment (christian-cg) [inline assets/icons/look-down.svg:2]: Are you indenting with 4 spaces?

File: css/home.css (hunk 6 above)
Comment (christian-cg) [reply css/home.css]: Yeah actually main-nav's was named main-nav__logo.

File: js/index.js [hunk 13]
```diff
@@ -46,3 +46,9 @@ const carrouselImage = document.querySelector(".carrousel__img");
 carrouselArrows.forEach(function(element) {
   element.addEventListener("click", carrousel);
 });
+/* Function about the arrow down or drop down wherever you want
+*/
+function goDown(){
+  let lookdown = document.getElementById('carrousel-section');//take the destini to scroll
```
Comment (christian-cg) [inline js/index.js:99]: I think you can place the description-comment in the previous line and it will make the code cleaner. What do you think?

File: css/home.css [hunk 14]
```diff
@@ -57,26 +58,46 @@
...
+  right: 0;
+  left: 0;
+  width: 33px;
+  position: absolute;
+  margin: auto;
 }
 
-.homepage-hero__headline h1 {
-  margin: 0;
+.homepage-hero__hdlp {
```
Comment (Frankruiz90) [inline css/home.css]: I do not know if this name can be defined like this

File: index.html [hunk 15]
```diff
@@ -43,23 +43,23 @@
...
+  <section class="carrousel-section" id="carrousel-section">
+
     <div class="container">
       <div class="info">
         <h1 class="info-title">Comodidades</h1>
         <hr class="info-line">
-        <p class="info-description">
-          Nuestro hostal te ofrece todas las comodidades para que vivas la mejor experiencia.
-        </p>
+          <p class="info-description">
```
Comment (josemariogutierrez) [inline index.html]: Fix indentation

**Comment by jusk02** [comment]
all of the classes with `hdlp` don't really make sense, even if hdlp is hostal donde las paisas, these classes should be fine using words like logo, headline, text, etc... depending on each case.

==========

Todas las clases con `hdlp` no tienen mucho sentido, aun siendo que hdlp es hostal donde las paisas, todas estas clases deberian funcionar con palabras como logo, headline, text, etc... segun sea cada caso.

**Comment by Frankruiz90** [comment]
image


the buttons on the part of the carousel were on the contrary are, they were good before uploading this commit, please check that you moved

File: index.html [hunk 16]
```diff
@@ -86,13 +86,12 @@ <h1 class="info-title">Comodidades</h1>
           <div class="service__icon">
             <img src="assets/icons/services/sunset.svg" alt="Sunset">
           </div>
-          <p class="service__description">Zona de playa privada</p>
```
Comment (camigomezdev) [inline index.html]: why did you remove this code, if this isn't part of your task?

**Review by jhonatanrojasbastidas — APPROVED** [review]

File: css/style.css [hunk 17]
```diff
@@ -350,8 +350,8 @@ video {
   margin-left: 14.5px;
```
Comment (josemariogutierrez) [inline css/style.css]: Spacing between dots are not right, change 14.5 to 11px. Also remove padding-left: 0%

**Review by josemariogutierrez — APPROVED** [review]
